addons/addon-base-raas/packages/base-raas-services/lib/indexes/indexes-service.js

```diff
@@ -116,9 +132,21 @@ class IndexesService extends Service {
     );
 
     // Validate input
-    const [validationService] = await this.service(['jsonSchemaValidationService']);
+    const [validationService, awsAccountsService] = await this.service([
+      'jsonSchemaValidationService',
+      'awsAccountsService',
+    ]);
     await validationService.ensureValid(rawData, updateSchema);
 
+    // Make sure the AWS Account actually exists
+    if (!_.isUndefined(rawData.awsAccountId)) {
+      try {
+        await awsAccountsService.mustFind(requestContext, { id: rawData.awsAccountId });
+      } catch (err) {
+        throw this.boom.badRequest(`Incorrect AWS Account ID provided`, true).cause(err);
```

Comment:
Same ` vs '   if this is not giving you linting issues, then we can leave it as is